src/main/java/jalau/at18/architects/ResourceCounter.java

```diff
@@ -0,0 +1,38 @@
+package jalau.at18.architects;
+
+import java.util.LinkedList;
+
+public class ResourceCounter {
+    private LinkedList<String> listOfResources;
+    private GrayCard grayCard;
```

Comment:
Attributes are not that extensible, instead of `LinkedList`, it might be better `List`, instead of `GrayCard`, it might be better `Card`. Open/Close and Dependency Inversion principles would be applied